frontend/packages/llama-stack-modular-ui/bff/internal/api/models_handler_test.go

```diff
@@ -0,0 +1,189 @@
+package api
+
+import (
+	"net/http"
+
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	"github.com/opendatahub-io/llama-stack-modular-ui/internal/integrations/llamastack"
+	"github.com/opendatahub-io/llama-stack-modular-ui/internal/models"
+)
+
+var _ = ginkgo.Describe("Models Handler", func() {
+	var ExpectedModels []models.Model
+	var ExpectedLLMModels []models.Model
+	var ExpectedEmbeddingModels []models.Model
+
+	ginkgo.BeforeEach(func() {
```

Comment:
instead of `BeforeEach`, just declare this as a var?  It's a good practice to have an AfterEach for every BeforeEach statement, so that we know what we spin up and what we spin down.